Bundle/BlogBundle/Entity/BlogCategory.php

```diff
@@ -187,7 +188,7 @@ public function getSlug()
      *
      * @param string $articles
      *
-     * @return PostPage
+     * @return $this
```

Comment:
You shouldn't do this, please prefer the ClassName, here: BlogCategory or self
https://stackoverflow.com/questions/6945564/should-i-use-return-self-this-or-the-current-class